site/content/docs/user/configuration.md

```diff
@@ -455,5 +455,47 @@ nodes:
         node-labels: "my-label3=true"
 {{< /codeFromInline >}}
 
+If you need more control over patching, strategic merge and JSON6092 patches can
+be used as well. These are specified using files in a directory, for example
+`./patches/kube-controller-manager.yaml` could be the following.
+
+{{< codeFromInline lang="yaml" >}}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-controller-manager
+  namespace: kube-system
+spec:
+  containers:
+  - name: kube-controller-manager
+    env:
+    - name: KUBE_CACHE_MUTATION_DETECTOR
+      value: "true"
+{{< /codeFromInline >}}
+
+Then in your kind YAML configuration use the following.
+
+{{< codeFromInline lang="yaml" >}}
+nodes:
+- role: control-plane
+  extraMounts:
+  - hostPath: ./patches
+    containerPath: /patches
+
+kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    patches:
+      directory: /patches
+{{< codeFromInline >}}
+
+Note the `extraMounts` stanza. `kind` runs kubeadm from a container in the node,
```

Comment:
nit: I think this is slightly mis-worded / confusing. For kind the node is a container, and we run kubeadm in the node-container, but not in a container "within" the node (like say, kube-proxy).

Reply:
Good point thanks. Here'a another try.